lazy-load.php

```diff
@@ -73,13 +76,17 @@ static function process_image( $matches ) {
 
 		$image_src = $old_attributes['src']['value'];
 
-		// Remove src and lazy-src since we manually add them
-		$new_attributes = $old_attributes;
-		unset( $new_attributes['src'], $new_attributes['data-lazy-src'] );
+        // Also retain the srcset, and sizes if present. The latter is to keep w3c validator happy (sizes requires srcset)
+        $lazy_srcset_attribute = ( ! empty( $old_attributes['srcset'] ) ) ? 'data-lazy-srcset="' . $old_attributes['srcset']['value'] . '"' : '';
+        $lazy_sizes_attribute = ( ! empty( $old_attributes['sizes'] ) ) ? 'data-lazy-sizes="' . $old_attributes['sizes']['value'] . '"' : '';
```

Comment:
Instead of creating the attribute here, we should just assign the value and build it as part of the HTML further down. Similar to how we're handling the `src` attribute. This helps us more clearly escape the attribute value.